### ISSUE TITLE
Telemetry sent without an instrumentation key

### ISSUE THREAD

**Issue by bryancrosby** [microsoft/ApplicationInsights-Home#451]
I recently installed the Application Insights Status monitor on a cloud-hosted VM. I wanted to see some detailed SQL command text from a console application, using the custom SDK.

In my application, which is a console application, I created an Application Insights resource in Azure and set the telemetry configuration to point to this key.

I'm seeing too much data on my subscription -- already over 5 GB -- how did the Application Insights Status monitor know to associate the data to my instrumentation key? I was expecting just a dozen or so SQL and HTTP calls, but I have over 1 million events. I do have other .NET apps that run on this VM (all console tasks), but I didn't do *anything* with Status Monitor other than just installing it and restarting the VM.

So, in summary:

1) Installed agent
2) Deployed console application to VM with a specific telemetry configuration pointed to a specific instrumentation key
3) Noticed other telemetry data from other .NET console applications that run on the same VM
4) Was not expecting other application's telemetry data other than the specific application I configured with a specific instrumentation key

Any idea how I can whitelist just this one console application?
